test/plugins/test_case_time_log_test.py

```diff
@@ -73,3 +77,22 @@ def test_json_reporter_reports(self):
                 json.loads(self.reporter.log_file.getvalue()),
                 json.loads(output_str),
             )
+
+    def test_test_case_results_reports(self):
+        tempdir = tempfile.mkdtemp()
+        temp_json_file = os.path.join(tempdir, 'tmp.json')
```

Comment:
this leaks the tempdir and tempfile, you should use a context manager to clean that up

You can peek at `test.discovery_failure_test.BrokenImportTestCase` for an example